### PR TITLE
`notifyOperatorInactivity` for Wallet Registry

### PR DESCRIPTION
**Pull request by pdyraga**
Refs https://github.com/keep-network/keep-core/issues/2839

The function lets the wallet signing group operators notify about operators who are
inactive in the group. Using this function, a majority of the wallet signing
group can decide about punishing specific group members who constantly fail
to do their job. If the provided claim is proved to be valid and signed by
a sufficient number of group members, operators of members deemed as inactive are
banned from sortition pool rewards for the duration specified by
`sortitionPoolRewardsBanDuration` parameter. The sender of the claim must be one
of the claim signers. This function can be called only for registered wallets.

TODO (in a separate PR):
- [ ] ETH reimbursement for `notifyOperatorInactivity`

### CODE REVIEW

File: solidity/ecdsa/contracts/libraries/Wallets.sol [hunk 1]
```diff
@@ -81,11 +81,24 @@ library Wallets {
         return self.registry[walletID].publicKeyX != bytes32(0);
     }
 
+    /// @notice Returns Keccak256 hash of the wallet signing group members
+    ///         identifiers array. Group members do not include operators
+    ///         selected by the sortition pool that misbehaved during DKG.
+    /// @param walletID ID of the wallet
+    /// @return Wallet signing group members hash
+    function getWalletMembersIdsHash(Data storage self, bytes32 walletID)
```
Comment (lukasz-zimnoch) [inline solidity/ecdsa/contracts/libraries/Wallets.sol:86]: Worth documenting that this method will return `bytes32(0)` if `walletID` refers to an unknown wallet.

File: solidity/ecdsa/contracts/libraries/Wallets.sol [hunk 2]
```diff
@@ -81,11 +81,24 @@ library Wallets {
...
+    }
+
     /// @notice Gets public key of a wallet with a given wallet ID.
     ///         The public key is returned in an uncompressed format as a 64-byte
     ///         concatenation of X and Y coordinates.
-    /// @param walletID ID of the wallet.
-    /// @return Uncompressed public key of the wallet.
+    /// @param walletID ID of the wallet
+    /// @return Uncompressed public key of the wallet
     function getWalletPublicKey(Data storage self, bytes32 walletID)
```
Comment (lukasz-zimnoch) [inline solidity/ecdsa/contracts/libraries/Wallets.sol:99]: Worth documenting that this method will revert if `walletID` refers to an unknown wallet.

**Review by lukasz-zimnoch — APPROVED** [review]
Just two non-blocking comments.

File: solidity/ecdsa/contracts/libraries/Wallets.sol (hunk 1 above)
Comment (pdyraga) [reply solidity/ecdsa/contracts/libraries/Wallets.sol:86]: Done in #2908, commit https://github.com/keep-network/keep-core/pull/2908/commits/b9898f4d6eace62316d9bb512a94559d7f4f96a0.

File: solidity/ecdsa/contracts/libraries/Wallets.sol (hunk 2 above)
Comment (pdyraga) [reply solidity/ecdsa/contracts/libraries/Wallets.sol:99]: Done in #2908, commit https://github.com/keep-network/keep-core/pull/2908/commits/b9898f4d6eace62316d9bb512a94559d7f4f96a0.

**Pull request merged**